### ISSUE TITLE
onion address, proxies down

### ISSUE THREAD

**Issue by vv01f** [RadicalMilitantLibrary/manual#16]
The mentioned onion service is down permanently for a while.
It was moved to a next generation onion service – that one is down to me as well.
Also the mentioned onion proxies are gone.

**Comment by Jotunbane** [comment]
Yeah sorry. Everything is down for now. Personal reasons prevent me from hosting anything right now. I am working on a solution, but we are still at the drawing board, and the paper is all but blank.

**Comment by vv01f** [comment]
@Jotunbane nobody blames you ;) write us if we can help, otherwise take your time.

this is just about how useful the manual is and it was directed to the last editor @ewa4boeker  of it ;) – assigned for clarification.
